etna/models/autoarima.py

```diff
@@ -57,7 +57,10 @@ class AutoARIMAModel(
     """
     Class for holding auto arima model.
 
-    Method ``predict`` can use true target values only on train data on future data autoregression
+    Method ``forecast`` can be used on ouf-of-sample data that goes after training data with a gap.
```

Comment:
I'm not really sure about adding this clarifications into documentation.

Reply:
Ok, let's do this in some other task. May be it better fits the documentation track.